iofog-agent-daemon/src/main/java/org/eclipse/iofog/pruning/DockerPruningManager.java

```diff
@@ -83,7 +83,7 @@ public void start() throws Exception {
         if (!isPruning) {
             long availableDiskPercentage = StatusReporter.getResourceConsumptionManagerStatus().getAvailableDisk() * 100 /
                     StatusReporter.getResourceConsumptionManagerStatus().getTotalDiskSpace();
-            if (availableDiskPercentage < Configuration.getAvailableDiskThreshold()) {
+            if (availableDiskPercentage < 90) {
```

Comment:
What is the magic number?

Reply:
Oh sorry I was debugging this locally. Pushed my local changes as well